main.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  metric_filters = var.create_metric_filter ? { for filter in var.metric_filters : filter.name => filter } : {}
+  metric_alarms = { for alarm in var.metric_alarms : alarm.alarm_name => alarm }
```

Comment:
Not sure why the variable name is being changed? This is still the metric-filter module, not a metric-alarm module?